satpy/etc/readers/lma_l2.yaml

```diff
@@ -0,0 +1,22 @@
+file_types:
+    mimicTPW2_comp:
```

Comment:
This filetype should probably be called something else not to get confused with mimic